README.md

```diff
@@ -811,6 +813,23 @@ RuboCop::RakeTask.new(:rubocop) do |task|
 end
 ```
 
+## Caching
+
+Large projects containing hundreds or even thousands of files can take
+a really long time to inspect, but RuboCop has functionality to
+mitigate this problem. A result caching mechanism is activated if the
+configuration parameter `AllCops: UseCache` is `true`, which it is by
+default. The command line option `--cache false` can be used to turn
+off result caching, thus overriding the configuration parameter. If
+`AllCops: UseCache` is set to `false` in the local `.rubocop.yml`,
+then it's `--cache true` that overrides the setting.
+
+The cache is stored in `.rubocop_cache/` under the current directory.
+There are subdirectories for each user, so as to minimize the risk of
+collisions and file curruption from simultaneous `rubocop`
+invocations. Git users should add `.rubocop_cache` to their
```

Comment:
@jonas054 any chance you could make the cache directory configurable in an option? CircleCI allows us to cache specific directories between runs (maybe Travis has something similar?). We do this now for rails asset caching - speeds up an asset compile from 2+ minutes to 10 seconds, and CircleCI stores the cache per-branch.

I don't think CircleCI will let us cache anything out of /tmp though, since I think it only reloads things under the project/user directory. I suppose we could always work around this by copying the cache directory, but that's less than ideal.

For reference, this is what our CircleCI config looks like to achieve this:
```yml
dependencies:
  pre:
    - mkdir -p public/assets
    - mkdir -p tmp/cache/assets
  post:
    - bundle exec rake assets:precompile:
        environment:
          RAILS_ENV: test
  cache_directories:
    - public/assets/
    - tmp/cache/assets
```